### PR TITLE
Mlshvch hometask 1

### PR DESCRIPTION
**Pull request by mlshvch**
I've completed Ruby-Lab hometask 1 with described requirements.

### CODE REVIEW

**Review by StalemateInc — DISMISSED** [review]
👍

A small side note: it’s a good practice to try and fix the styleguide offences locally before pushing to the remote to a) minimize the noise in PR; b) reduce the amount of spam from Hound. Hound is simply a bot which uses your Rubocop config, which means that you can check for offences by yourself. Just install the Rubocop locally (`gem install rubocop:1.22.1`) and make a habit of running it before opening the PR. This way the amount of commits with style fixes can be reduced drastically.

More tips like this one you can find in #general channel in Student’s Lab Slack workspace.

**Review by ramantechart — CHANGES_REQUESTED** [review]
Fix your commit history.
Hound fixes should be wrapped in a one commit

File: mlshvch/hometask-1/hello_world.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# !/usr/bin/env ruby
```
Comment (ramantechart) [inline mlshvch/hometask-1/hello_world.rb]: This won't make the file executable.
Try to run `$ ./hello_world.rb` in the hometask folder.

**Review by ramantechart — CHANGES_REQUESTED** [review]

File: mlshvch/hometask-1/hello_world.rb (hunk 1 above)
Comment (StalemateInc) [reply mlshvch/hometask-1/hello_world.rb]: Or, to be precise, that's not _the only_ thing to make it executable.

Comment (mlshvch) [reply mlshvch/hometask-1/hello_world.rb]: Do you mean adding information about "chmod +x /mlshvch/hometask-1/hello_world.rb" into README.md or sth else?

And as I've understood from previous comments, I need to revert state of branch to "Create hello_world.tb file" commit by means of "git revert"?

Comment (ramantechart) [reply mlshvch/hometask-1/hello_world.rb]: @StalemateInc @mlshvch this part especially was done with typo which makes me thinking it wasn't tested locally

Comment (ramantechart) [reply mlshvch/hometask-1/hello_world.rb]: @mlshvch
`This won't make the file executable.` means your file is not executable.
```
➜  hometask-1 git:(mlshvch-hometask-1) ✗ ./hello_world.rb
./hello_world.rb: line 5: puts: command not found
```
You have a typo in line 3.

Speaking of commit history, you've got a bunch of options.
Some are easy to do, some are pretty complex, but I'm going to suggest the simplest one.

In my opinion for this particular PR based on changes you've made, your commit history should look like:
```
- Create hello_world.rb file
- Make hello_world.rb file executable
- Added frozen_string comment to hello_world.rb file
- Fixed comments in hello_world.rb file
```

or a bit simpler:
```
- Create hello_world.rb file
- Make hello_world.rb file executable
- Fix Hound violations
```

But as soon as you haven't completed `executable` part of the task, in your case, I'd suggest to do next:
- complete the task
- run rubocop locally in order to avoind Hound violations
- squash commits in one locally and it'll 1 commit history
- push changes to the branch

**Review by ramantechart — APPROVED** [review]

**Review by StalemateInc — APPROVED** [review]

**Pull request merged**